week5.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<p>Looks like this section got copied and pasted twice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>